src/ansible_navigator/data/default_ee_dockerfile

```diff
@@ -0,0 +1 @@
+FROM quay.io/ansible/creator-ee:v0.9.2
```

Comment:
You MUST keep the name `Dockerfile` or dependabot will not work. There is no way to configure custom filenames.

Reply:
https://github.com/dependabot/dependabot-core/compare/v0.67.1...v0.67.2

although I'm not sure how to test it

Reply:
You cannot, if the path does not work, we make another PR to fix it.